classes/game.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at, publish_date: Time.now, archived: nil, id: rand(1..1000))
+    super(publish_date, archived: archived, id: id)
+    @multiplayer = multiplayer || 'Unknown'
+    @last_played_at = last_played_at || Time.now
+  end
```

Comment:
Can you please add properties according to the diagram. You added `publish_date` which ain't part of them
![image](https://user-images.githubusercontent.com/102610715/196654654-8704cdbc-4f53-447e-8620-43d0d4a6a483.png)